src/platform/silabs/EFR32/wifi/wfx_notify.cpp

```diff
@@ -187,3 +189,44 @@ void wfx_ip_changed_notify(int got_ip)
         chip::DeviceLayer::PlatformMgr().UnlockChipStack();
     }
 }
+
+/**************************************************************************************
+ * @fn  void wfx_retry_interval_handler(bool is_disconnection_event, uint16_t retryJoin)
+ * @brief
+ *      Based on condition will choose specific interval.
```

Comment:
Should say smth like "Based on condition will delay for a certain period of time"